lib/models/device.coffee

```diff
@@ -88,9 +90,33 @@ getDeviceModel = (deps, opts) ->
 		if semver.lt(version, minVersion)
 			throw new Error("Incompatible supervisor version: #{version} - must be >= #{minVersion}")
 
+	###*
+	# @summary Get Dashboard device URL for a device
+	# @function getDashboardUrl
+	#
+	# @param {Object} options - options
+	# @param {String} options.appId - Application id
+	# @param {String} options.deviceId - Device id
+	#
+	# @returns {String} - Dashboard URL
+	# @throws Exception if either appId or deviceId are not non-empty strings
+	#
+	# @example
+	# dashboardUrl = resin.models.device.getDashboardUrl({ appId: '012345', deviceId: '678901' })
+	###
+	exports.getDashboardUrl = getDashboardUrl = (options = {}) ->
+		isNonEmptyString = (str) ->
+			isString(str) and str.length > 0
+
+			for key in [ 'appId', 'deviceId' ]
+				throw new Error("#{key} should be a non-empty string") if not isNonEmptyString(options[key])
+
+			return "#{settings.get('dashboardUrl')}/apps/#{options.appId}/devices/#{options.deviceId}/summary"
+
 	addApplicationName = (device) ->
```

Comment:
The method name is not appropriate anymore, please give it a better name, like `addExtraInfo` or smth like that